app-src/player-router/.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "4.1"
```

Comment:
Looks like I just have to drop Travis – the tests aren't headless right now, so can't be run by Travis.